main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/codegangsta/cli"
+)
+
+const (
+	version = "0.2"
+	usage   = `Open Container Initiative hypervisor-based runtime
+
+runv is a command line client for running applications packaged according to
+the Open Container Format (OCF) and is a compliant implementation of the
+Open Container Initiative specification.  However, due to the difference
+between hypervisors and containers, the following sections of OCF don't
+apply to runV:
+    Namespace
+    Capability
+    Device
+    "linux" and "mount" fields in OCI specs are ignored
+
+The current release of "runV" supports the following hypervisors:
+    KVM (QEMU 2.0 or later)
+    Xen (4.5 or later)
+    VirtualBox (Mac OS X)
+
+After creating a spec for your root filesystem, you can execute a container
+in your shell by running:
+
```

Comment:
these messages are already on runv readme. usage is used to show how to use this program, isn't it?